igvm/vm.py

```diff
@@ -457,6 +457,10 @@ def build(self, run_puppet=True, debug_puppet=False, postboot=None):
 
             if run_puppet:
                 self.run_puppet(clear_cert=True, debug=debug_puppet)
+                if rebuild:
+                    self.run_puppet(clear_cert=True, debug=debug_puppet)
```

Comment:
how about `self.run_puppet(clear_cert=rebuild, debug=debug_puppet)`

Reply:
Shiny, I like it.